documentation/docs/09-seo.md

```diff
@@ -0,0 +1,70 @@
+---
+title: SEO
+---
+
+The most important aspect of SEO is to create high-quality content that is widely linked to from around the web. However, there are a few technical considerations for building sites that rank well.
+
+### Zero config SEO benefits
+
+#### SSR
+
+By default, SvelteKit does server-side rendering. While some search engines can index client-side content, not all do. For those that do, there will often be a larger delay between publishing content and having it indexed with client-side rendering compared to server-side rendering. Thus, it is recommended that you leave SSR on for best results. SvelteKit offers highly configurable rendering that will allow you to do things like [dynamic rendering](https://developers.google.com/search/docs/advanced/javascript/dynamic-rendering) where you serve server-side rendered pages to search engines and client-rendered pages to your users. This is not necessarily something that we recommend for a number of reasons such as the increased possibility of having breakages visible only to search engines, the difficulty in creating an exhaustive whitelist of search engine user agents, and a worse experience for users with JavaScript disabled. However, the fact that it would be fairly easy to setup does show off SvelteKit's abilities rather well.
```

Comment:
"However, the fact that it would be fairly easy to setup does show off SvelteKit's abilities rather well." seems like a weird thing to put here, and sounds like something that someone writing a comparison blog post between different frameworks would write, not something that would go in official documentation.